### DISCUSSION TITLE
Deta space login error  in Windows 7

### DISCUSSION THREAD

**Discussion by suantro** [deta/space-docs#152] [Q&A] [answered]
I try to login in Win7. The error is:

>space login
To authenticate the Space CLI with your Space account, generate a new access token in your Space settings and paste it below:

←[?25lError: prepare console: set console mode: The parameter is incorrect.

I'v got this error in Cmd console, PowerShell and in VSCode terminal. Can I create some file in some folder like this https://github.com/orgs/deta/discussions/46 to login in Win7?

**Comment by pomdtr** [comment, marked as answer]
Hey, the library we use for interactive inputs is not working on windows 7. You can use the non-interactive version of the command instead:

```
# git bash
cat "<my-token>" | space login --with-token

# windows console
type "<my-token>" | space login --with-token

# windows powershell
Get-Content "<my-token>" | space login --with-token
```
